gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -1198,7 +1198,7 @@ public <T> T fromJson(JsonReader reader, Type typeOfT) throws JsonIOException, J
    * <p>Unlike the other {@code fromJson} methods, no exception is thrown if the JSON data has
    * multiple top-level JSON elements, or if there is trailing data.
    *
-   * <p>The JSON data is parsed in {@linkplain JsonReader#setLenient(boolean) lenient mode},
+   * <p>//todo: update javadoc The JSON data is parsed in {@ linkplain JsonReader#setLenient(boolean) lenient mode},
```

Comment:
The build fails because there is a whitespace between `@` and `linkplain`.

Should be: `@linkplain ...`